subsys/bluetooth/host/Kconfig

```diff
@@ -338,9 +338,11 @@ if BT_SMP
 config BT_PRIVACY
 	bool "Privacy Feature"
 	help
-	  Enable local Privacy Feature support. This makes it possible
+	  Enable Privacy Feature support. This makes it possible
 	  to use Resolvable Private Addresses (RPAs).
 
+	  Disabling this will remove the capability to resolve private addresses.
```

Comment:
```suggestion
	  Enable Privacy Feature support. This makes it possible
	  to generate and use Resolvable Private Addresses (RPAs).

	  Disabling this will remove the capability to resolve private addresses.
```